src/test/java/de/mediathekview/mserver/crawler/zdf/ZdfEntryDtoTest.java

```diff
@@ -0,0 +1,68 @@
+package de.mediathekview.mserver.crawler.zdf;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import org.junit.Test;
+
+public class ZdfEntryDtoTest {
+
+  @Test
+  public void equalsTestNull() {
+    final ZdfEntryDto target = new ZdfEntryDto("test", "hallo");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "hallo" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)